sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -176,23 +178,25 @@ class SdkSynchronizer private constructor(
             fastestServerFetcher: FastestServerFetcher,
             fetchExchangeChangeUsd: UsdExchangeRateFetcher,
             preferenceProvider: PreferenceProvider,
+            torClient: TorClient,
```

Comment:
Technically there's no need to store this at the top level, if you've already passed a `torClient.isolatedClient()` handle into everywhere that needs it (as long as every instance of `TorClient` does eventually have `TorClient.dispose()` called).

But it costs almost nothing to store here, and it means it's around for future `torClient.isolatedClient()` calls if ever you need to shut down and then restart some inner component while keeping the overall `Synchronizer` running. So this is fine.